src/client/containers/SettingsModal.tsx

```diff
@@ -17,6 +17,7 @@ import { Option } from '@/components/SettingsModal/Option'
 import { Shortcut } from '@/components/SettingsModal/Shortcut'
 import { NotesSortKey } from '@/utils/enums'
 import { NotesSortByOption } from '@/components/SettingsModal/SortByOption'
+import { NotesDirectionTextOption } from '@/components/SettingsModal/DirectionTextOption'
```

Comment:
There is no reason to create a new component for this; it should be combined with the `SortByOption` to create a generic component that can handle the two, almost identical use-cases.